swingset/src/main/java/com/nqadmin/swingset/utils/SSComponentInterface.java

```diff
@@ -330,36 +366,63 @@ default RowSet getRowSet() {
 //		removeSSComponentListener();
 //	}
 
+//	/**
+//	 * Removes listener for Document if SwingSet component is a JTextComponent.
+//	 * <p>
+//	 * Implementation of removeSSComponentListener() can just call this method when
+//	 * the component is a JTextComponent.
+//	 */
+//	default void removeDocumentListener() {
+//		getSSCommon().removeDocumentListener();
+//	}
+	
 	/**
-	 * Remove the listener detecting changes in value for the current component.
-	 * SSCommon will manage any bound RowSet listeners.
-	 * <p>
-	 * Generally the developer will need to add an inner class and corresponding
-	 * data member that implements the appropriate listener for the JComponent
-	 * involved (e.g., ItemListener for a class extending JCheckBox, ChangeListener
-	 * for a class extending JSlider, DocumentListener for a class extending
-	 * JTextField, etc.).
-	 * <p>
-	 * If the component is JTextComponent then the implementation can simply call
-	 * removeSSDocumentListener()
+	 * Indicates if the components RowSet listener is added/enabled.
+	 * 
+	 * @param _component SwingSet component to be checked for a RowSet listener
+	 *
+	 * @return true if the component's RowSet listener is added/enabled, otherwise false
 	 */
-	void removeSSComponentListener();
+	static boolean isRowSetListenerAdded(SSComponentInterface _component) {
+		return _component.getSSCommon().isRowSetListenerAdded();
+	}
```

Comment:
Why not
```
boolean isRowSetListenerAdded() {
    return getSSCommon().isRowSetLIstenerAdded();
```

And I guess there's a few others. Probably something I'm missing.

Reply:
Struggled with this. It's an interface so I can't make a method final, only default so that can invite the user to override. Making it static, they can't. I want to be as restrictive as possible. Could probably even eliminate the static methods.

Reply:
Got it, makes sense. It makes the intention explicit.